src/pages/ExamsPage/components/AllowanceListActions.jsx

```diff
@@ -0,0 +1,91 @@
+import { PropTypes } from 'prop-types';
+import {
+  ActionRow,
+  Button,
+  Icon,
+  IconButtonWithTooltip,
+  ModalDialog,
+  useToggle,
+} from '@openedx/paragon';
+import { DeleteOutline, EditOutline } from '@openedx/paragon/icons';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import messages from '../messages';
+import { useDeleteAllowance } from '../hooks';
+
+const DeleteModal = (isOpen, onCancel, onDelete, formatMessage) => (
+  <ModalDialog
+    title="delete allowance"
+    isOpen={isOpen}
+    onClose={onCancel}
+    variant="default"
+    hasCloseButton
+    isFullscreenOnMobile
+  >
+    <ModalDialog.Header>
+      <ModalDialog.Title>
+        {formatMessage(messages.deleteAllowanceHeader)}
+      </ModalDialog.Title>
+    </ModalDialog.Header>
+    <ModalDialog.Body>
+      <p>{formatMessage(messages.deleteAllowanceBody)}</p>
+    </ModalDialog.Body>
+    <ModalDialog.Footer>
+      <ActionRow>
+        <ModalDialog.CloseButton variant="tertiary" onClick={onCancel}>
+          {formatMessage(messages.deleteAllowanceCancel)}
+        </ModalDialog.CloseButton>
+        <Button variant="primary" onClick={onDelete}>
```

Comment:
Do we want this to be a stateful button? Just curious about error handling in the case that the request isn't successful

Reply:
As it's written now, won't it close regardless of whether the call is successful or not?

Reply:
I must have been looking at an older version of this, I see that it's now set to close only on success!

Reply:
yeah added this morning but this conversation reminded me I should explicitly test it as well.